app/components/WalletDetails.tsx

```diff
@@ -23,19 +23,47 @@ export const WalletDetails = () => {
   const { smartWalletBalance, allBalances, refreshBalance } = useBalance();
 
   const { user } = usePrivy();
+
   const { fundWallet } = useFundWallet({
-    onUserExited: ({ fundingMethod, balance, chain, address }) => {
-      refreshBalance();
-      trackEvent("funding_completed", {
-        wallet: "smart_wallet",
-        address,
-        network: chain.name,
-        fundingType: fundingMethod,
-        amount: balance,
-      });
+    onUserExited: ({ fundingMethod, chain }) => {
+      // NOTE: This is an inaccurate way of tracking funding status
+      // Privy doesn't provide detailed funding status information
+      // Available variables in onUserExited: address, chain, fundingMethod, balance
+      // Limitations:
+      // 1. fundingMethod only indicates user selected a method, not if funding completed
+      // 2. User can select method and cancel, but it still records as "completed"
+      // 3. No way to track funding errors
+      // 4. balance is returned as bigint and doesn't specify token type
+      // 5. No webhook or callback for actual funding confirmation
+      if (fundingMethod) {
+        refreshBalance();
+        trackEvent("Funding Completed", {
+          "Funding type": fundingMethod,
+          Amount: "Not available on Privy",
+          Network: chain.name,
+          Token: "USDC", // privy only supports USDC
```

Comment:
this isn't correct... token is customizable

refer to `handleFundWallet` in `TransactionForm.tsx`

although to customize in this component we need to have a preliminary modal that allows selection of asset